_posts/2018-11-27-tide-middleware-evolution.md

```diff
@@ -71,7 +71,7 @@ pub struct RequestContext<'a, Data> {
 
 impl<'a, Data: Clone + Send> RequestContext<'a, Data> {
     /// Consume this context, and run remaining middleware chain to completion.
-    pub fn next(mut self) -> FutureObj<'a, Response>;
+    pub fn next(self) -> FutureObj<'a, Response> { ... }
```

Comment:
I tried to follow the way rustdoc shows us. Not exactly same, of course!

I thought `mut` is unnecessary as we're omitting the method body here.